### DISCUSSION TITLE
評価点数方法について

### DISCUSSION THREAD

**Discussion by putcho01** [jphacks/F_2203#39] [Ideas]
## 名前どうするか
- GPA
- ポイント
- 推し熱度、、、

## 評価方法どうするか

**Comment by putcho01** [comment]
取得できるデータ
- 投稿数
- お気に入り アーティスト数
- 投稿の位置情報
-

**Comment by cyacyamaru007** [comment]
採点基準は「貢献度」で評価したい。
・情報をより多く入力した人を評価する制度！
　◦投稿数に応じて評価
　◦アーティスト数

**Reply by putcho01** [reply]
良さそう。

**Reply by cyacyamaru007** [reply]
![Uploading Screenshot_20221020_160503_jp.linknetwork.anitrip.jpg…]()
